src/securejoin.rs

```diff
@@ -499,6 +499,12 @@ pub(crate) async fn handle_securejoin_handshake(
 
             inviter_progress!(context, contact_id, 300);
 
+            // for setup-contact, make Alice's one-to-one chat with Bob visible
+            // (secure-join-information are shown in the group chat)
+            if !join_vg {
+                ChatId::create_for_contact(context, contact_id).await?;
```

Comment:
we could also add an info-message to the chat here, sth. as "Bob has just scanned your QR code, waiting for Bob to verify ..." (suggestions welcome - but we should maybe better do and discuss that in another pr :)